src/main/java/technical/TaskList.java

```diff
@@ -0,0 +1,303 @@
+    toReply[0] = "Here are the tasks in your list.";
+    for (int i = 0; i < taskList.size(); ++i) {
+      toReply[i + 1] = String.format("%d. %s", i + 1, taskList.get(i));
+    }
+    Ui.reply(toReply);
+  }
+
+  public static void justAddedComment() {
+    Ui.reply(new String[] {"Successfully added the following task",
+        taskList.get(taskList.size() - 1).toString(),
+        String.format("You now have %d tasks in the list.",
+        taskList.size())});
+  }
+
+  /**
+   * Append a functional.Todo to the todoList.
+   *
+   * @param arguments The command arguments.
+   */
+  public static void todo(String[] arguments) throws IOException {
+    StringBuilder todoName = new StringBuilder();
+    for (int i = 1; i < arguments.length; ++i) {
+      if (todoName.length() != 0) {
+        todoName.append(' ');
+      }
+      todoName.append(arguments[i]);
+    }
+    if (todoName.length() == 0) {
+      Ui.reply("Please include a name");
+      return;
+    }
+    taskList.add(new Todo(todoName.toString()));
+    SaveFile.addData(taskList.get(taskList.size() - 1).toData());
+    justAddedComment();
+  }
+
+  /**
+   * Append a functional.Deadline to the todoList.
+   *
+   * @param arguments The command arguments.
+   */
+  public static void deadline(String[] arguments) throws IOException {
+    StringBuilder deadlineName = new StringBuilder();
+    StringBuilder deadlineDeadlineString = new StringBuilder();
+    boolean byFlagRead = false;
+    for (int i = 1; i < arguments.length; ++i) {
+      if (arguments[i].equals("/by") && !byFlagRead) {
+        byFlagRead = true;
+        continue;
+      }
+      if (byFlagRead) {
+        if (deadlineDeadlineString.length() != 0) {
+          deadlineDeadlineString.append(' ');
+        }
+        deadlineDeadlineString.append(arguments[i]);
+      } else {
+        if (deadlineName.length() != 0) {
+          deadlineName.append(' ');
+        }
+        deadlineName.append(arguments[i]);
+      }
+    }
+    if (deadlineName.length() == 0 || deadlineDeadlineString.length() == 0) {
+      Ui.reply(new String[]{"Format the command as follows:",
+          "deadline <deadline name> /by <deadline>"});
+      return;
+    }
+    LocalDateTime deadlineDeadline;
+    try {
+      deadlineDeadline = LocalDateTime
+          .parse(deadlineDeadlineString.toString(),
+          DateTimeFormatter.ofPattern("yyyy/M/d H:m:s"));
+    } catch (DateTimeParseException e) {
+      Ui.reply("Please format your date as"
+          + " \"year/month/date hour/minute/second\" (24 hour format).");
+      return;
+    }
+    taskList.add(new Deadline(deadlineName.toString(), deadlineDeadline));
+    SaveFile.addData(taskList.get(taskList.size() - 1).toData());
+    justAddedComment();
+  }
+
+  /**
+   * Append a functional.Event to the todoList.
+   *
+   * @param arguments The command arguments.
+   */
+  public static void event(String[] arguments) throws IOException {
```

Comment:
method names should be verbs in camel case, so maybe you can consider naming it addEvent, addTodo, etc